ui/component/viewers/videoViewer/internal/videojs-events.jsx

```diff
@@ -86,7 +95,21 @@ const VideoJsEvents = ({
         userId,
         uri,
         this, // pass the player
-        bitrateAsBitsPerSecond
+        bitrateAsBitsPerSecond,
+        isLivestreamClaim
+      );
+    } else {
+      // populates data for watchman, sends prom and matomo event
+      analytics.videoStartEvent(
+        claimId,
+        0,
+        playerPoweredBy,
+        userId,
+        uri,
+        this, // pass the player
+        // $FlowFixMe
+        this.vhs?.playlists.media().attributes.BANDWIDTH,
```

Comment:
Just an observation:
On the RT stream, oddly 480p and 720p results in the same value of `BANDWIDTH: 200000`.  Only 1080p changes the value.  Normal?

Reply:
Will check this out, but it may only change once it buffers for now.